tests/conftest.py

```diff
@@ -299,7 +299,7 @@ def net_cleanup(self):
             fds = re_space.split(l)
             if len(fds) > 1:
                 pname = fds[1].rstrip(":")
-                m = re.compile("(eth|lo|Bridge|Ethernet)").match(pname)
+                m = re.compile("(eth|lo|Bridge|Ethernet|vEthernet)").match(pname)
```

Comment:
check this commit. https://github.com/Azure/sonic-swss/commit/c74dc6079f49814ef290b65ed3e80b9cb27b310b#diff-484462fced51d1a06b1d93b4a44dd535

vEthernet is no longer used in the vs docker.

Reply:
ok. then these interfaces shall not be created?

Reply:
no, they should not. in what cases do you see them?